source/includes/harvest/_users.md

```diff
@@ -143,11 +143,11 @@ id | The ID of the user whose job permissions to retrieve
 
 An array of objects of the following:
 
-Parameter | Description
---------- | -----------
-id | The ID representing the job permission
-job_id | The ID of the job for the job permission
-user_role_id | The ID of the user role for the job permission
+Parameter | Type | Description
+--------- | ----------- | ------------
+id | integer | The ID representing the job permission
+job_id | integer | The ID of the job for the job permission
+user\_role\_id | integer | The ID of the user role for the job permission
```

Comment:
This feels a little confusing to me. What if it was something like 

+job_id | integer | The ID of the job to be added or removed
+user\_role\_id | integer | The ID of the user role to be assigned for this job